src/types.rs

```diff
@@ -98,12 +119,75 @@ pub struct Assignment {
     pub name: String,
 }
 
-#[derive(Responder)]
-#[response(status = 400)]
-pub struct BadRequest(pub &'static str);
+// Input type for /hosts/stats endpoint
+// Data schema in collection `holoports_status`
+#[derive(Serialize, Deserialize, Clone)]
+#[serde(crate = "rocket::serde")]
+#[serde(rename_all = "camelCase")]
+pub struct HostStats {
```

Comment:
This new struct HostStats supersedes both Host struct and HostSummary struct. All the code should be updated to reflect that change.